dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -24,6 +24,26 @@ _get_cluster_status(){
     echo $(rosa list clusters -o json | jq -r '.[] | select(.name == '\"$1\"') | .status.state')
 }
 
+_wait_for_cluster_ready(){
+    echo "INFO: Waiting about 3 hour for cluster on ready status"
+    # 180 iterations, sleeping 60 seconds, 3 hours of wait
+    for i in `seq 1 180` ; do
+        CLUSTER_STATUS=$(_get_cluster_status $1)
+        if [ ${CLUSTER_STATUS} == "ready" ] ; then
+            sleep 120 # Wait 120 seconds to allow OCM to really finish the cluster installation
+	    echo "INFO: Cluster on ready status, dumping installation logs..."
+            rosa logs install -c $1
```

Comment:
@morenod one other request. Can you run a rosa describe cluster as well after the install? Has some useful information in there that might help if we need to debug.

Reply:
done

Reply:
ok, now I think it is really done